plugin-template

```diff
@@ -217,6 +218,28 @@ def main():
                         """
         ),
     )
+    parser.add_argument(
+        "--add-version-ci-update-branches",
+        metavar="VERSION_BRANCH",
+        action="store",
+        help=textwrap.dedent(
+            """\
+                            Add specified version to the ci_update_branches before templating.
+
+                        """
+        ),
+    )
+    parser.add_argument(
+        "--replace-previous-version-ci-update-branches",
+        action="store_true",
+        help=textwrap.dedent(
+            """\
+                            Used with --add-version-ci-update-branches, replaces the previous
+                            version in ci_update_branches with the new version.
+
+                        """
+        ),
+    )
```

Comment:
If i'm not mistaken, the script should be able defer this option from the template_config.